modules/widgets/search.php

```diff
@@ -552,39 +552,46 @@ private function maybe_render_sort_javascript( $instance, $order, $orderby ) {
-					var orderByDefault = '<?php echo 'date' === $orderby ? 'date' : 'relevance'; ?>',
-						orderDefault   = '<?php echo 'ASC' === $order ? 'ASC' : 'DESC'; ?>',
-						widgetId       = decodeURIComponent( '<?php echo rawurlencode( $this->id ); ?>' ),
-						searchQuery    = decodeURIComponent( '<?php echo rawurlencode( get_query_var( 's', '' ) ); ?>' ),
-						isSearch       = <?php echo (int) is_search(); ?>;
-
-					var container = $( '#' + widgetId + '-wrapper' ),
-						form = container.find('.jetpack-search-form form'),
-						orderBy = form.find( 'input[name=orderby]'),
-						order = form.find( 'input[name=order]'),
-						searchInput = form.find( 'input[name="s"]' );
-
-					orderBy.val( orderByDefault );
-					order.val( orderDefault );
-
-					// Some themes don't set the search query, which results in the query being lost
-					// when doing a sort selection. So, if the query isn't set, let's set it now. This approach
-					// is chosen over running a regex over HTML for every search query performed.
-					if ( isSearch && ! searchInput.val() ) {
-						searchInput.val( searchQuery );
-					}
-
-					searchInput.addClass( 'show-placeholder' );
-
-					container.find( '.jetpack-search-sort' ).change( function( event ) {
-						var values  = event.target.value.split( '|' );
-						orderBy.val( values[0] );
-						order.val( values[1] );
-
-						form.submit();
-					});
+			var jetpackSearchModuleSorting = function() {
+				var orderByDefault = '<?php echo 'date' === $orderby ? 'date' : 'relevance'; ?>',
+					orderDefault   = '<?php echo 'ASC' === $order ? 'ASC' : 'DESC'; ?>',
+					widgetId       = decodeURIComponent( '<?php echo rawurlencode( $this->id ); ?>' ),
+					searchQuery    = decodeURIComponent( '<?php echo rawurlencode( get_query_var( 's', '' ) ); ?>' ),
+					isSearch       = <?php echo (int) is_search(); ?>;
+
+				var container = document.getElementById( widgetId + '-wrapper' ),
+					form = container.querySelector( '.jetpack-search-form form' ),
+					orderBy = form.querySelector( 'input[name=orderby]' ),
+					order = form.querySelector( 'input[name=order]' ),
+					searchInput = form.querySelector( 'input[name="s"]' ),
+					sortSelectInput = container.querySelector( '.jetpack-search-sort' );
+
+				orderBy.value = orderByDefault;
+				order.value = orderDefault;
+
+				// Some themes don't set the search query, which results in the query being lost
+				// when doing a sort selection. So, if the query isn't set, let's set it now. This approach
+				// is chosen over running a regex over HTML for every search query performed.
+				if ( isSearch && ! searchInput.value ) {
+					searchInput.value = searchQuery;
+				}
+
+				searchInput.classList.add( 'show-placeholder' );
+
+				sortSelectInput.addEventListener( 'change', function( event ) {
+					var values  = event.target.value.split( '|' );
+					orderBy.value = values[0];
+					order.value = values[1];
+
+					form.submit();
```

Comment:
This select works as expected:

![form-do](https://user-images.githubusercontent.com/4063887/76279162-4d7bfb80-6253-11ea-9904-54942896c6dc.gif)